sklearn/linear_model/ridge.py

```diff
@@ -1096,7 +1095,7 @@ def __init__(self, alphas=(0.1, 1.0, 10.0),
                  fit_intercept=True, normalize=False, scoring=None,
                  cv=None, gcv_mode=None,
                  store_cv_values=False):
-        self.alphas = np.asarray(alphas, dtype=np.float64)
+        self.alphas = np.asarray(alphas)
```

Comment:
Oh, I see that this was already done in master. These days we would avoid such validation.